api/routes/index.js

```diff
@@ -36,7 +40,8 @@ module.exports = function (passport) {
 				User.findOneAndUpdate({ 'username': user.username }, { 'status': false }, function (err, user) {
 					if (err) { return res.send(err); }
 					console.log('Deactivating user: ', user.username);
-					return res.send({ user: user, message: 'Successfully deactivated user' });
+					res.setHeader("Content-Type", "application/json");
```

Comment:
`res.json(...)` should already be setting the header to `"Content-Type": "application/json"`